prepare_benchmark

```diff
@@ -20,6 +21,10 @@ if __name__ == '__main__':
         logger.error('Your python version does not match the requirements.')
         sys.exit()
 
+    if which('psql') is None:
+        print('ERROR: Could not find psql. Is it installed? Is PATH setup properly?')
+        sys.exit()
+
```

Comment:
I propose to put this somewhere central since you use it again below.

Reply:
Also: please add a check for "recode" if the benchmark is TPC-DS.

Reply:
Added checks.py. Done